cores/esp32/esp32-hal-rgb-led.c

```diff
@@ -15,7 +29,40 @@ void neopixelWrite(uint8_t pin, uint8_t red_val, uint8_t green_val, uint8_t blue
     return;
   }
 
-  int color[] = {green_val, red_val, blue_val};  // Color coding is in order GREEN, RED, BLUE
+#if defined RGB_BUILTIN_LED_COLOR_ORDER_CHANGE
```

Comment:
maybe remove the `_CHANGE` suffix. `RGB_BUILTIN_LED_COLOR_ORDER` is more readable and makes more sense

Reply:
Done.